arrays/Task1.js

```diff
@@ -0,0 +1,37 @@
+const numbers1 = [2, -5, 0, 7, -3, 0, 10, -8, 'Hello', -8, 5]
+const numbers2 = [5, 9, 0 ,-10, 7, 0, -1, -99]
+
+let positiveCount
```

Comment:
initialise value here
```js
let positiveCount = 0
let negativeCount = 0
let zeroCount = 0
```

and remove it from function

Reply:
Ah, I see, you call it with different sets

Reply:
then do not use global variables. Create it in the function

Reply:
it was supposed to do without a function but if you use function then you can return result values from function